marimo/_cli/serve/commands.py

```diff
@@ -0,0 +1,49 @@
+from pathlib import Path
+
+import click
+
+from marimo._cli.print import echo, green
+from marimo._server.utils import find_free_port
+
+
+@click.command(
+    help="""Serve an WebAssembly HTML notebook.
+
+This will start a simple HTTP server to serve the HTML file with
+the proper headers required for marimo's WebAssembly.
+
+This is useful for testing the notebook in the browser,
+but is not recommended for production.
+Instead, you should use a proper web server to serve the notebook.
+
+Example:
+
+  \b
+  * marimo serve notebook.html
+
+"""
+)
+@click.argument("filename", required=True, type=click.Path(exists=True))
+def serve(filename: str) -> None:
```

Comment:
While this is neat, I would prefer to not add `serve` to the CLI right now.

Since this code snippet is so small and doesn't depend on any marimo internals (the port can be hardcoded instead), I would instead suggest adding the code snippet to the docs, or even publishing it as a script/gist on GitHub and recommending it be run with `uv run`

Also, `serve` and `run` are close to synonyms, so the terminology is a bit confusing.

Reply:
If you feel strongly though can be convinced to keep it. Just seems simpler to not include since it doesn't use marimo internals, and is readily accomplished in just a few lines of code or a GitHub gist.

Reply:
Could a one liner similar to this work?

https://stackoverflow.com/questions/8058793/single-line-python-webserver

Reply:
i started with the one-liner in the docs, but figured it wasn't clear. i will remove this though